### PR TITLE
Fixed #8782 - User has incorrect rights

### LINKED ISSUE #8782: Cloning Zio-QuickStarts in Overview Guides Results in Fatal if User has Incorrect Rights
Hello , the link in [quickstart guide for restfule web service](https://zio.dev/guides/quickstarts/restful-webservice/#running-the-example) uses ssh login. it'll be easier to just clone via _Https_ ? Thanks

image

### PR DESCRIPTION
**Pull request by Yummy-Yums**

### CODE REVIEW

**Comment by Yummy-Yums** [comment]
#8782

**Comment by Yummy-Yums** [comment]
@khajavi please re-trigger CI build , the previous run was successful

**Comment by Yummy-Yums** [comment]
@khajavi please re-trigger CI build , the previous run was successful

File: docs/contributing-to-documentation.md [hunk 1]
```diff
@@ -47,8 +47,15 @@ ZIO contributors are encouraged to use this approach for introducing new documen
 1. First, we need to clone the ZIO project on our machine:
 
 ```bash
-git clone git@github.com:zio/zio.git
-cd zio
+$ git clone git@github.com:zio/zio.git
+$ cd zio
+```
+
+if the above throws an error , try cloning via https:
```
Comment (khajavi) [inline docs/contributing-to-documentation.md]: I prefer using only one section for cloning the GitHub repo. We can use the https scheme and omit the alternative section.

**Comment by Yummy-Yums** [comment]
@khajavi regarding _make-a-zio-application-configurable.md_ , can you confirm if that's the right link to clone , i can't find it under your github 

image

**Comment by khajavi** [comment]
@Yummy-Yums No, that is obsolete and should be fixed!
Currently, project is one of the subprojects of `zio-quickstarts`:
https://github.com/zio/zio-quickstarts/tree/master/zio-quickstart-restful-webservice-configurable-app

**Review by khajavi — DISMISSED** [review]

**Review by khajavi — APPROVED** [review]
Perfect! Thank you!

**Pull request merged**